src/core-plugins/CoreCommands.ts

```diff
@@ -55,13 +78,13 @@ export default class CoreCommands extends Plugin {
 
         this.commands.registerCommand({
             name: "debug",
-            description: "Posts debug info",
+            description: "Post Debug info.",
```

Comment:
Why did you change all these to statements? I would think an action makes more sense